frontend/src/lib/components/LemonSelectMultiple/LemonSelectMultiple.tsx

```diff
@@ -18,13 +18,16 @@ export interface LemonSelectMultipleOptionItem extends LemonSelectMultipleOption
 export type LemonSelectMultipleOptions = Record<string, LemonSelectMultipleOption>
 
 export interface LemonSelectMultipleProps {
+    selectClassName?: string
```

Comment:
I believe this was intentionally omitted by @benjackwhite so that it's harder to do hacky edits. And actually it turns it in the case of this component this is not needed, because when we take away the `property-filters-property-value` class (exactly the type of hacky solution which really belongs to the design system level), it appears we don't need `selectClassName`! Pushed this change.

Reply:
Ahh, I was wrong about this being redundant currently, because then things break in cohort definition UI without this, because that place _also_ mixes this half-lemon and other full-ant components, but in a different way from insight query definition UI, so… arrrgh. Let's leave as is for now, I'll be making a new ant-free `LemonSelectMultiple` anyway in the next few weeks.